lib/Tmdb/Model/Tv/Episode.php

```diff
@@ -164,7 +164,9 @@ public function getAirDate()
      */
     public function setAirDate($airDate = null)
     {
-        if (!$airDate instanceof DateTime && $airDate !== null) {
+        if ($airDate === '') {
+            $airDate = null;
+        } elseif (!$airDate instanceof DateTime && $airDate !== null) {
```

Comment:
@ptheofan think we are better off changing the check to not empty, would you mind checking if this might occur in different models?
```suggestion
        } elseif (!$airDate instanceof DateTime && !empty($airDate)) {
```

Reply:
Couldn't agree more. Didn't go with empty in the first place as a lot of people find it to be tricky for junior devs. :)